perkpack-frontend/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Put everything in here into our top level .gitignore

Reply:
Fixed.